staging/src/k8s.io/client-go/tools/cache/reflector_test.go

```diff
@@ -435,6 +435,41 @@ func TestReflectorWatchListPageSize(t *testing.T) {
 	}
 }
 
+func TestReflectorNotPaginatingNotConsistentReads(t *testing.T) {
```

Comment:
can we also separately test behavior when WatchListPageSize is set?

Reply:
The test above already does that - `TestReflectorWatchListPageSize`

Reply:
Aah - I guess you're asking about case for explicit RV (that test was using RV="0").
Set RV explicitly in that test.